spec/features/hub/edit_13614c_spec.rb

```diff
@@ -391,7 +391,7 @@
 
       expect(intake.cv_other_income_cb_yes?).to eq true
 
-      expect(intake.cv_p2_notes_comments).to eq "Hello"
+      expect(intake.cv_p2_notes_comments).to eq "additional income from banana stand"
```

Comment:
should we see "Other money received during the year includes:" text anywhere in the feature specs?